src/bot/Slackbot.py

```diff
@@ -24,14 +25,39 @@ def get_message():
         message = requests.get(url, headers=header, params=payload)
         return "message"
 
-    def send_message_to_slander_api(self,message):
+    # Send message and get slander level score from slander level api
+    def send_message_to_slander_api(self, message):
         url = 'http://3.143.237.69/kusorep/score/'
         message = message
         param = {'msg': message}
         res = requests.get(url, params=param)
         slander = res.json()["body"]["kusoripu_score"]
         return slander
 
+    # Get measurement result from slander level api and check whether it is a slander
+    def get_measurement_result(self):
+
+        # get slander level from slander level api
+        message = self.get_message()
+        measurement_result_from_slander_api = self.send_message_to_slander_api(message)
+        measurement_result = measurement_result_from_slander_api[0]
```

Comment:
Maybe the slander that returns from above function is equal measurement_result

Reply:
your function return the value type is list, I need to use float value.